apps/web/src/app/test-kyc/page.tsx

```
@@ -0,0 +1,150 @@
+'use client';
+
+import { useState } from 'react';
+import { useKYCWebSocket, type KYCUpdate } from '@/hooks/use-kyc-websocket';
+import { Card } from '@/components/ui/card';
+import { Button } from '@/components/ui/button';
+import { Badge } from '@/components/ui/badge';
+
+export default function TestKYCPage() {
+  const [userId] = useState('test-user-' + Math.random().toString(36).slice(2, 7));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)